src/NServiceBus.Core.Analyzer.Tests/Helpers/DiagnosticVerifier.cs

```diff
@@ -24,13 +25,13 @@ public abstract class DiagnosticVerifier
 
         protected abstract DiagnosticAnalyzer GetAnalyzer();
 
-        protected Task Verify(string source, params DiagnosticResult[] expectedResults) => Verify(new[] { source }, expectedResults);
+        protected Task Verify(string source, DiagnosticResult[] expectedResults, CancellationToken cancellationToken = default) => Verify(new[] { source }, expectedResults, cancellationToken);
 
-        async Task Verify(string[] sources, params DiagnosticResult[] expectedResults)
+        async Task Verify(string[] sources, DiagnosticResult[] expectedResults, CancellationToken cancellationToken)
```

Comment:
I do not like this part. I think I'd rather editorconfig this assembly out of the analyzer.